E/edfplusdcnv/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder
+
+# Collection of pre-build edfplusdcnv binaries
+name = "edfplusdcnv"
+edfplusdcnv_ver = "1.0.0"
+version = VersionNumber(edfplusdcnv_ver)
+
+sources = [
+    ArchiveSource("https://www.teuniz.net/edfplusd-converter/edfplusdcnv_100.tar.gz", "eeb6033a2f21dc052852e3dd779da7d368f65d88e45b7e40fa14f4bb514e62b9"; unpack_target = "x86_64-linux-gnu"),
```

Comment:
With `unpack_target = "x86_64-linux-gnu"` you're saying "call the directory `x86_64-linux-gnu`", so you either don't do this or in the build script below you install the file `${target}/edfplusdcnv`:
```suggestion
    ArchiveSource("https://www.teuniz.net/edfplusd-converter/edfplusdcnv_100.tar.gz", "eeb6033a2f21dc052852e3dd779da7d368f65d88e45b7e40fa14f4bb514e62b9"),
```